_im-plugin/ism/error-prevention/api.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: ISM error prevention API
+parent: ISM error prevention
+nav_order: 10
+---
+
+# ISM Error prevention API
+
+The ISM Error Prevention API allows you to enable ISM error prevention, as well as checking the validation status and message.
+
+## Enable error prevention validation
+
+Error Prevention Validation is configured by setting the `plugins.index_state_management.validation_service.enabled` parameter.
```

Comment:
active voice suggestion: 
"You can configure error prevention validation by setting the . . . "
Or 
"To configure error prevention validation, set the . . . "